spec/features/audit/report/progress_spec.rb

```diff
@@ -1,72 +1,102 @@
-    before(:each) do
-      content_items = create_list(
-        :content_item,
-        3,
-        document_type: "transaction",
-        allocated_to: me,
-        primary_publishing_organisation: my_organisation,
-      )
+  scenario "Displaying the number of items needing improvement/not needing improvement" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    and_one_content_item_needs_improvement_and_one_does_not
+    when_i_visit_the_report_page
+    then_i_see_that_50_percent_of_content_needs_improvement
+    and_i_see_that_50_percent_of_content_does_not_need_improvement
+  end
 
-      create(:passing_audit, content_item: content_items[0])
-      create(:failing_audit, content_item: content_items[1])
-    end
+  scenario "Audit status filter is not present" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    when_i_visit_the_report_page
+    then_i_do_not_see_audit_status_filter
+  end
 
-    scenario "Displaying the number of items included in the audit" do
-      visit audits_report_path
+  def then_i_see_the_report_includes_all_three_content_items
+    expect(@audit_report_page).to have_report_section(text: '3 Content items')
+  end
 
-      expect(page).to have_content("3 Content items")
+  def then_i_see_the_report_includes_no_content_items
+    expect(@audit_report_page).to have_report_section(text: '0 Content items')
+  end
 
-      select "Anyone", from: "allocated_to"
-      select "Guide", from: "document_type"
-      click_on "Apply filters"
+  def then_i_do_not_see_audit_status_filter
+    expect(@audit_report_page).to_not have_report_section(text: 'Audit Status')
+  end
 
-      expect(page).to have_content("0 Content items")
+  def given_i_am_an_auditor_belonging_to_an_organisation_with_three_content_items
+    organisation = create(:organisation)
+    user = create(:user, organisation: organisation)
+    @content_items = create_list(
+      :content_item,
+      3,
+      document_type: "transaction",
+      allocated_to: user,
+      primary_publishing_organisation: organisation,
+    )
+  end
 
-      select "Transaction", from: "document_type"
-      click_on "Apply filters"
+  def and_one_content_item_needs_improvement_and_one_does_not
+    create(:passing_audit, content_item: @content_items[0])
+    create(:failing_audit, content_item: @content_items[1])
+  end
 
-      expect(page).to have_content("3 Content items")
-    end
+  def and_two_items_have_been_audited
+    create(:passing_audit, content_item: @content_items[0])
+    create(:failing_audit, content_item: @content_items[1])
+  end
 
-    scenario "Displaying the number of items audited/not audited" do
-      visit audits_report_path
+  def when_i_visit_the_report_page
+    @audit_report_page = ContentAuditTool.new.audit_report_page
+    @audit_report_page.load(content_id: Content::Item.last.content_id)
```

Comment:
Again, not sure why we need a content ID for the report page?

Reply:
√